web-sourcecode/src/components/SumaClient.vue

```diff
@@ -560,17 +564,77 @@ $tippy_textcolor: white;
 .activityGroup {
   border: 1px solid DarkGray;
   border-radius: 5px;
-  padding: 2px 5px 5px;
+  // padding: 2px 5px 5px;
```

Comment:
We should delete this and not comment it out.

Reply:
will do.